packages/ui/src/components/vuestic-components/va-tabs/VaTabs.demo.vue

```diff
@@ -378,14 +344,41 @@
       >
         <template #tabs>
           <va-tab
-            v-for="title in ['One', 'Two', 'Three', 'Four']"
```

Comment:
No need to move back, but generally it's fine to keep stuff like this in demos.
You don't really have to refactor these or change even.

Actually I would've preferred something like this:
```vue
<va-tab>One</va-tab>
<va-tab>Two</va-tab>
....
```
unless you're testing adding/removing tabs.